doc/PREFERENCE.md

```diff
@@ -0,0 +1,48 @@
+## Mark Text Preference
+
+#### General
+
+| Key                 | Type    | Default Value | Description                                                                                                                                                |
+| ------------------- | ------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| autoSave            | Boolean | ture          | Automatically save the content being edited. option value: true, false                                                                                     |
+| autoSaveDelay       | Number  | 3000          | How long do you want to save your document(ms)?3000 ~10000                                                                                                 |
+| titleBarStyle       | String  | csd           | The title bar style. the native option will result in a standard gray opaque title bar. `csd`(macOS only), `custom`, `native`                              |
+| openFileInNewWindow | Boolean | false         | true, false                                                                                                                                                |
+| aidou               | Boolean | true          | Enable aidou. Optional value: true, false                                                                                                                  |
+| fileSortBy          | String  | modified      | Sort files in opened folder by `created`time, modified time and title.                                                                                     |
+| startUp             | String  | lastState     | The action after Mark Text startup, open the last edited content, open the specified folder or blank page, optional value: `lasteState`, `folder`, `blank` |
+| language            | String  | en            | The language Mark Text use.                                                                                                                                |
+
+#### Editor
+
+| Key                    | Type    | Defaut             | Description                                                                                                                                                           |
+| ---------------------- | ------- | ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| fontSize               | Number  | 16                 | Font size in pixels. 12 ~ 20                                                                                                                                          |
```

Comment:
I think we should increase the maximum font size to `32` or so.

Reply:
I updated it, there still other problem that it can not change the heading font size.